Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -465,6 +465,22 @@ lemma ΓSpec_adjunction_homEquiv_eq {X : Scheme.{u}} {B : CommRingCat} (φ : B 
   simp_rw [← ΓSpecIso_inv_ΓSpec_adjunction_homEquiv φ]
   simp
 
+lemma toSpecΓ_ΓSpec_adjunction_homEquiv {X : Scheme.{u}} {B : CommRingCat} (φ : B ⟶ Γ(X, ⊤)) :
```

Comment:
Is this just `ΓSpecIso_inv_ΓSpec_adjunction_homEquiv`?
What's the right spelling here? I'm thinking we should just remove `toSpecΓ`.

Reply:
Good catch, this was accidentally re-introduced by merging.